tests/AbstractTestCase.php

```diff
@@ -18,4 +18,21 @@ protected function getServiceProviderClass($app)
     {
         return BugsnagServiceProvider::class;
     }
+
+    /**
+     * Get the value of the given property on the given object.
+     *
+     * @param object $object
+     * @param string $property
+     *
+     * @return mixed
+     */
+    protected function getProperty($object, $property)
+    {
+        $propertyAccessor = function ($property) {
+            return $this->{$property};
+        };
+
+        return $propertyAccessor->bindTo($object, $object)($property);
+    }
```

Comment:
I'd usually avoid things like this but it's kind of hard to tests these configuration options using the methods that exist in [the `Bugsnag\Configuration` class](https://github.com/bugsnag/bugsnag-php/blob/master/src/Configuration.php), but I'm happy to change the approach